### ISSUE TITLE
Forslag til plan

### ISSUE THREAD

**Issue by jonathf** [realistforeningen/bearstock#15]
Her er noen feature requests fra "media". Hvert punkt skal skje på forskjellige tidspunkter.

1. Vanlig utvalg, ingen prisjusteringer og karantene.
2. Skru på karantene 5 minutter.
3. Starte børsen, reduser utvalg til ca. 20 alternativer
4. "On-the-fly" endre noen brukeres stats til å midlertidig visere f.eks. 50 % bedre stats enn hva de egentlig har.
5. Streik: reduser vareutvalget midlertidig til ca 6 typer
6. Ny vare introduseres, med lav start-pris
7. Krakk: To-tre vare dumper i pris
8. Justere karantene-tid underveis

Si gjerne ifra om hva som lar seg og ikke lar seg gjøre.

**Comment by eistaa** [comment]
I think most of it should be doable. Except for the quarantine (2. and 8.), and boosting the stats of some users (4.), I think we did all last time.

@jonathf Regarding the quarantine: You mean quarantining users? As quarantine of stocks should already be handled by hiding the product.

**Comment by jonathf** [comment]
Yes quarantining the user. As in: They just bought a beer, now they can not buy one for the next 5 minutes.

Siden note: Do we have a plan for informing the user about the quarantine their currently in. Like a timer saying "3min 23sec until new purchases is allowed"? I imagine lots of frustration among drunk people if they do not have a full overview.

**Comment by eistaa** [comment]
That's on the roadmap, it was one of the things we talked about adding after some user bought 9-10 beers at once last time.

Yea, I think using a timer of some sort is the way to go to notify them.

**Comment by jonathf** [comment]
Excellent! This is all I need for my part.
